### PR TITLE
Travis CI builds

### PR DESCRIPTION
**Pull request by orien**
Get Travis CI builds working.

https://travis-ci.org/envato/event_sourcery/

### CODE REVIEW

**Comment by orien** [comment]
BuildKite files deleted in e5f8f86.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+sudo: false
+language: ruby
+rvm:
+  - 2.2
+  - 2.3
+  - 2.4
+before_install:
+  - gem install bundler
```
Comment (joesustaric) [inline .travis.yml]: So where do we execute the specs?

Comment (twe4ked) [inline .travis.yml]: I think by default it will run `rake` which I'm assuming is the specs.

Comment (orien) [inline .travis.yml]: TravisCI will run `rake` by default. https://docs.travis-ci.com/user/languages/ruby/#Default-Test-Script

Comment (joesustaric) [inline .travis.yml]: ah ok cool, I think I remember that now.. Thanks

**Pull request merged**